src/main/kotlin/net/djvk/fireflyPlaidConnector2/transactions/TransactionConverter.kt

```diff
@@ -186,17 +200,29 @@ class TransactionConverter(
          * Don't pass in [plaidUpdatedTxs] here because we're not going to try to update transfers for now
          *  because it's more complexity than I want to deal with, and I haven't seen any Plaid updates in the wild yet
```

Comment:
This feels like the sort of thing where some financial institutions never use it, and some sometimes use it.

Maybe it would make sense to open an issue to capture demand for this. If we see non-empty `plaidUpdatedTxs`, emit a log entry that links to the issue and asks the user if they'd like to help contribute back by commenting and letting us know a few things like what financial institution sent the update.

Reply:
Sounds good to me.

Reply:
https://github.com/dvankley/firefly-plaid-connector-2/issues/106